Source/RxCBPeripheral.swift

```diff
@@ -47,6 +47,10 @@ class RxCBPeripheral: RxPeripheralType {
         return peripheral.identifier
     }
 
+    var objectId: Int {
+        unsafeBitCast(peripheral, to: Int.self)
```

Comment:
I was thinking about `UInt(ObjectIdentifier(peripheral))` as it more proper way
for doing this? Look at: https://github.com/apple/swift/blob/adc54c8a4d13fbebfeb68244bac401ef2528d6d0/stdlib/public/core/ObjectIdentifier.swift